module/x/gravity/keeper/keeper.go

```diff
@@ -27,7 +27,7 @@ import (
 type Keeper struct {
 	StakingKeeper          types.StakingKeeper
 	EthereumEventProcessor interface {
-		Handle(sdk.Context, types.EthereumEvent) error
+		Handle(sdk.Context, *Keeper, types.EthereumEvent) error
```

Comment:
imo this should be a `HandleFn` instead of an interface

Reply:
I just changed it into a keeper method.